horace_core/sqw/@SQWDnDBase/mask_points.m

```diff
@@ -72,7 +72,13 @@
 end
 
 % Check input parameters
-ndim=length(win.data.pax);
+if isa(win, 'sqw')
+    data = win.data;
+else
+    data = win;
+end
+
+ndim = length(data.pax);
 
```

Comment:
There is method `data.NUM_DIMS`  (hidden, do not know why it is hidden -- not my design) which is intended for this. 
Probably it should be used here.